recipe/meta.yaml

```diff
@@ -21,8 +21,7 @@ requirements:
     - python
     - pip
     - cython
-    - numpy 1.14.*  # [py==37]
-    - numpy 1.12.*  # [py!=37]
```

Comment:
Just a question for my understanding here: conda-forge dropped building against numpy 1.12? (pandas still supports it)